content/millennium/r4/financial/general/charge-item.md

```diff
@@ -212,6 +212,61 @@ The `ETag` response header indicates the current `If-Match` version to use on su
 
 The common [errors] and [OperationOutcomes] may be returned.
 
+## Operation: charge-item-modify
+
+<%= beta_tag(action: true) %>
+
+Modifies an existing ChargeItem resulting in a newly created ChargeItem.
+
+    POST /ChargeItem/$modify
+
+_Implementation Notes_
+
+* Only the body fields mentioned below are supported. Unsupported fields will be ignored.
```

Comment:
Are we really ignoring unsupported fields?

Reply:
based on discussions with Keifer and Alex updated wording as it fails when we pass unsupported fields
https://github.com/cerner/fhir.cerner.com/pull/780/commits/0878c6c8ccb78566c363cdb2ff6a03fe5a21c8f9#diff-03f941e4ceb2839bfe0b49a5cc913e04919560e289603ae7b08b875f86ec568dR225